src/PeerPoolMgr.h

```diff
@@ -18,6 +18,30 @@ class HttpRequest;
 class CachePeer;
 class CommConnectCbParams;
 
+/// CodeContext for cache_peer related classes
+class DetailedCodeContext : public CodeContext
+{
+public:
+    typedef RefCount<DetailedCodeContext> Pointer;
+
+    DetailedCodeContext(const char *gist, const SBuf &detail, const MasterXaction::Pointer &mx) : gist_(gist),
```

Comment:
Here and elsewhere, please use natural language "items: item1, item2" formatting for labeled lists:

```suggestion
    DetailedCodeContext(const char *gist, const SBuf &detail, const MasterXaction::Pointer &mx): gist_(gist),
```

Reply:
Done.